### ISSUE TITLE
Address SVG icons not showing on homepage

### ISSUE THREAD

**Issue by debjudy** [GSA-TTS/tts.gsa.gov#214]
SVG icons not showing on latest update to site - example https://federalist-a2423046-fe43-4e75-a2ef-2651e5e123ca.sites.pages.cloud.gov/preview/gsa-tts/tts.gsa.gov/debjudy-info-session-yml/

 Image shows what is expected 
![image](https://github.com/user-attachments/assets/c020e86f-93c6-47ab-af66-9fed30d435d5)

**Comment by wesley-dean-gsa** [comment]
I tried looking for the solution last week but I couldn't figure it out.  I thought that maybe the `color` attribute on the icons was the same as the background or `fill` but I didn't make much progress in that regard.  I also noticed that when viewed outside of the context of the web page are a pleasant shade of blue; however, when they were displayed on the page, they were shades of gray.  This led me to believe that we're setting the color somehow, likely through some CSS.
